bundles/org.openhab.binding.samsungtv/README.md

```diff
@@ -191,7 +191,7 @@ This input allows you to set the internal art mode state from an external source
 
 ### artLabel
 
-`artlabel` is a String channel that receives the *intenal* lable of the artwork displayed. This will be something like `MY_0010` or `SAM-0123`. `MY` means it's art you uploaded, `SAM` means its from the Samsung art gallery.
+`artlabel` is a String channel that receives the _intenal_ lable of the artwork displayed. This will be something like `MY_0010` or `SAM-0123`. `MY` means it's art you uploaded, `SAM` means its from the Samsung art gallery.
```

Comment:
`intenal` is a typo of `internal`?

Reply:
```suggestion
`artlabel` is a String channel that receives the _internal_ label of the artwork displayed. This will be something like `MY_0010` or `SAM-0123`. `MY` means it's art you uploaded, `SAM` means its from the Samsung art gallery.
```